pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
> Previously, an empty termination type on the route was consider non-secure anyway. The router should ignore the certs with an empty termination field unless something has changed.

We want to tighten validation to prevent users from creating borked routes with a bunch of useless values, but that would make existing data in etcd invalid, which brings its own problems with it.